### PR TITLE
Homework.07.Ansible.Cruize

### PR DESCRIPTION
**Pull request by AndreiAkhrymenko**

### CODE REVIEW

**Review by pluhin — CHANGES_REQUESTED** [review]
Please use one template for all hosts

**Comment by AndreiAkhrymenko** [comment]
1. made 1 template for CFGs VS 4 files
2. added few loops for repeated tasks

File: aakhrymenko/07.Ansible.Cruize/Ansible/roles/test_role1_nginx/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    src: template.conf                         
+    dest: "/etc/nginx/sites-available/{{ item.name }}.conf"    
+    owner: root  
+    group: root  
+    mode: '0644' 
+  with_items: "{{var_virtual_hosts}}"
+  when: (ansible_os_family == 'RedHat' and "centos" in item.name) or
+        (ansible_os_family == 'Debian' and "ubuntu" in item.name)
+
+- name: Make Link site1
```
Comment (pluhin) [inline aakhrymenko/07.Ansible.Cruize/Ansible/roles/test_role1_nginx/tasks/main.yaml:91]: Should be on task with template for site config

**Review by pluhin — CHANGES_REQUESTED** [review]

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**